CONTRIBUTING.md

```diff
@@ -62,3 +62,27 @@ We use [SassDoc] to document Bourbon. [Annotations] should be ordered:
 
 [SassDoc]: http://sassdoc.com
 [Annotations]: http://sassdoc.com/annotations
+
+## Goverance
```

Comment:
Shall we move this whole section right up below `# Contributing`, replacing that first paragraph?